src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -518,12 +516,8 @@ private static void HookupHistory(ModelBuilder modelBuilder)
             .HasIndex(e => new { e.ResourceEntityId, e.FromStateVersion });
 
         modelBuilder
-            .Entity<NonFungibleIdDataHistory>()
-            .HasIndex(e => new { e.NonFungibleIdDataId, e.FromStateVersion });
-
-        modelBuilder
-            .Entity<NonFungibleIdStoreHistory>()
-            .HasIndex(e => new { e.NonFungibleResourceEntityId, e.FromStateVersion });
+            .Entity<NonFungibleIdHistory>()
```

Comment:
I think it may be still worth to name this table "data_history" or "content_history" as it represents well... the data/content of the NFID. Similarly "location_history" will hold the NFID location etc.